sdk/core/azure-core/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ add_library (
   src/credentials/credentials.cpp
   src/credentials/policy/policies.cpp
   src/http/policy.cpp
+  src/http/policy_retry.cpp
```

Comment:
`retry_policy` or `policy_retry`, which is better?

Reply:
Vote favor in `retry_policy`.

Reply:
I know it is robotized language, but I like how it follows `policy.cpp` alphabetically, so I would keep it BUT now I recollect that we have file naming guidelines, and most likely I need to name it the same way as class is named, I don't remember the conclusion there.

Reply:
Done.